core/src/main/java/com/linecorp/armeria/server/healthcheck/DiskMemoryHealthChecker.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 LY Corporation
+ *
+ * LY Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.healthcheck;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+import java.io.File;
+
+/**
+ * A {@link HealthChecker} that reports as unhealthy
+ * when the target free disk space exceeds a file disk usable space.
+ * For example:
+ * <pre>{@code
+ * final DiskMemoryHealthChecker diskMemoryHealthChecker = HealthChecker.ofDisk(100, new File("/tmp"));
+ *
+ * // Returns false if a file disk usable memory space is less than 100 bytes,
+ * // or true if a file disk usable memory space is greater than or equal to 100 bytes.
```

Comment:
```suggestion
 * final DiskMemoryHealthChecker diskMemoryHealthChecker = HealthChecker.ofDisk(0.8, new File("/tmp"));
 *
 * // Returns false if a file disk usable memory space is less than 80%.
 * // or true if a file disk usable memory space is greater than or equal to 80%.
```
Can you update doc properly too here?

Reply:
Oh, I missed it. Thanks.